README.md

```diff
@@ -88,6 +88,9 @@ This module exposes a single function `download` which takes the same arguments
 * **number_sample_per_shard** the number of sample that will be downloaded in one shard (default *10000*)
 * **save_additional_columns** list of additional columns to take from the csv/parquet files and save in metadata files (default *None*)
 * **timeout** maximum time (in seconds) to wait when trying to download an image (default *10*)
+* **max_format_tries** maximum amount of YouTube formats to try (disacrding slow downloading formats)
```

Comment:
typo

Reply:
can you remove the youtube word ? it's at least general to yt-dlp, and probably we can reuse the option for other things too